### DISCUSSION TITLE
My enrolment letter shows that I am a distance student but it is being rejected

### DISCUSSION THREAD

**Discussion by Kris-Martin** [community/community#54322] [GitHub Education]
### Hi everyone,

Hi,

I am having difficulty proving my status as a student. Firstly it won't let me upload a pdf or screenshot it forces me to use the camera which is difficult to manoeuvre and focus. Secondly I am studying online and my enrolment letter shows this but I am still being auto-rejected. Also for some reason Github is not aware of what student emails from my University look like. This process is unnecessarily difficult and not at all accessible. How do I confirm my enrolment or how do I request assistance?

**Comment by samus-aran** [comment]
Hi @Kris-Martin  👋
We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by Kris-Martin** [reply]
The rejection is because it says my campus is not located near me and I'm not listed as distance. My enrolment confirmation lists me as an online student which is the same thing but is being rejected. Also GitHub has the incorrect format of student emails which is why I've had to try dealing with this system in the first place.
